jenkins/JenkinsFile-rebuild-docker-images

```diff
@@ -125,7 +125,7 @@ pipeline {
               ).trim()
 
               currentBuild.displayName = "${TVM_CURRENT_SHORT_REF}"
-              TVM_DOCKER_TAG = "${TIMESTAMP_TAG}-${TVM_CURRENT_SHORT_REF}"
+              TVM_DOCKER_TAG = "${TIMESTAMP_TAG}-${TVM_CURRENT_SHORT_REF}${params.tvm_docker_tag_suffix}"
```

Comment:
can you add a default to the param so the file runs if the param is not given?

Reply:
So I defined the parameter within the Jenkinsfile and added an empty default. Will the job fail to build if the parameter is undefined in the job configuration, or will the default be used?

Reply:
i am pretty sure the default will get used, let's try.

Reply:
can we actually make this a prefix? then we won't mix in staging with production

Reply:
yup--fixed